src/index.html

```diff
@@ -4,9 +4,72 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--start-day-sun">
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
```

Comment:
Remove empty lines in this case
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```